index.js

```diff
@@ -31,7 +60,15 @@ function betweenExtremes() {}
  * Example: "A new month"
  * .- / -. . .-- / -- --- -. - ....
  */
-function morseCodeTranslator() {}
+function morseCodeTranslator(message, dictionary) {
+  morseCodeArray = [];
+  message = message.replaceAll(" ", "");
```

Comment:
Excellent use of replaceAll;

As a stretch see if you can solve this using a `regular epxression`. 

Regexes - as we call them - are fantastic ways to manipulate strings